From: Kenneth D'souza <kdsouza@redhat.com>
Date: Mon, 21 Jan 2019 11:51:59 +1000
Subject: [PATCH] CIFS: Show locallease in /proc/mounts for cifs shares mounted
 with locallease feature.
Git-commit: c8b6ac1a9d1fb26adaeaafc3fb872e6713dfeaf7
Patch-mainline: v5.2-rc1
References: bsc#1144333

Missing parameter that should be displayed in the mount list

Reviewed-by: Ronnie Sahlberg <lsahlber@redhat.com>
Signed-off-by: Kenneth D'souza <kdsouza@redhat.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Acked-by: Paulo Alcantara <palcantara@suse.de>
---
 fs/cifs/cifsfs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/fs/cifs/cifsfs.c b/fs/cifs/cifsfs.c
index 877174761efb..54c065ada4de 100644
--- a/fs/cifs/cifsfs.c
+++ b/fs/cifs/cifsfs.c
@@ -483,6 +483,8 @@ cifs_show_options(struct seq_file *s, struct dentry *root)
 		seq_puts(s, ",seal");
 	if (tcon->nocase)
 		seq_puts(s, ",nocase");
+	if (tcon->local_lease)
+		seq_puts(s, ",locallease");
 	if (tcon->retry)
 		seq_puts(s, ",hard");
 	else
-- 
2.22.0

